bitnami/thanos/templates/query/ingress-grpc.yaml

```
@@ -18,19 +18,21 @@ spec:
     - host: {{ $query.ingress.grpc.hostname }}
       http:
         paths:
-          - path: /
-            backend:
-              serviceName: {{ template "common.names.fullname" . }}-query
-              servicePort: grpc
+          - path: {{ $query.ingress.path }}
+            {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
+            pathType: {{ $query.ingress.pathType }}
+            {{- end }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (printf "%s-%s" (include "common.names.fullname" .) "query") "servicePort" "http" "context" $)  | nindent 14 }}
```

Is this a typo? Same below.

```suggestion
            backend: {{- include "common.ingress.backend" (dict "serviceName" (printf "%s-%s" (include "common.names.fullname" .) "query") "servicePort" "grpc" "context" $)  | nindent 14 }}
```